src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -52,10 +53,6 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <!--
-      We check in the Release / Production build of blazor.*.js, but not the Debug builds. Consequently the former is always available to embed, the latter is only available
-      if Web.JS was built locally. Use the Debug build when available and building in Debug configuration.
-    -->
```

Comment:
This comment is still true?

Reply:
You told me to remove it: https://github.com/aspnet/AspNetCore/pull/12638#discussion_r307980166? Am I missing something?

Reply:
I was super correct, but it's no longer stale since that change was reverted. I guess follow your ❤️